client.go

```diff
@@ -273,6 +275,16 @@ type HTTPClient interface {
 // Keep this unexported so consumers of the package can't make changes to it.
 var defaultHTTPClient HTTPClient = newDefaultHTTPClient()
 
+type retryPolicy struct {
+	MaxDelay   time.Duration
+	MaxRetries int
+}
+
+var defaultRetryPolicy = retryPolicy{
```

Comment:
I think you should have a space between `retryPolicy` and `{`? Have you run `go fmt`?

Reply:
Yes. I have. In Go the bracket comes right after the struct name: 

```go
package main

import "fmt"

type Vertex struct {
	X int
	Y int
}

func main() {
	fmt.Println(Vertex{1, 2})
}
```
[Source](https://go.dev/tour/moretypes/2)